frontend/javascripts/oxalis/view/right-border-tabs/comment_tab/comment_tab_view.js

```diff
@@ -426,18 +431,27 @@ class CommentTabView extends React.Component<PropsWithSkeleton, CommentTabState>
                   />
                   <InputComponent
                     value={activeCommentContent}
-                    disabled={activeNodeMaybe.isNothing}
+                    disabled={isEditingDisabled}
+                    title={
+                      !this.props.allowUpdates
```

Comment:
Minor, feel free to ignore, but usually it's easier for a reader to parse this kind of condition if the "!" is dropped and the if/else is swapped. Same for the condition in line 449.